SonarQube.Client.Tests/SonarQubeService_GetHotspotAsync.cs

```diff
@@ -58,7 +58,10 @@ public async Task GetHotspot_Response_From_SonarQube()
     ""key"": ""java:S4787"",
     ""name"": ""rule-name"",
     ""securityCategory"": ""others"",
-    ""vulnerabilityProbability"": ""LOW""
+    ""vulnerabilityProbability"": ""LOW"",
+    ""riskDescription"": ""risk desc"",
+    ""vulnerabilityDescription"": ""vuln desc"",
+    ""fixRecommendations"": ""fix rec""
```

Comment:
Helpfully, the API also returns the three rule help fields, so I've included them too.